scientific_details_of_algorithms/linear_learner_multiclass_classification/linear_learner_multiclass_classification.ipynb

```diff
@@ -87,7 +87,8 @@
     "from sklearn.model_selection import train_test_split\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # unzip teh raw dataset
</pre><p><code>teh</code>  -&gt; <code>the</code>  :) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3176/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>